Task 6/script.js

```diff
@@ -19,3 +19,21 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30 },
   { id: '9', name: 'Daniel Cane', age: 51 },
 ];
+
+const getUserAverageAge = (arr = []) => {
+  const { sum, count } = arr.reduce((acc, val) => {
+     let { sum, count } = acc;
+     sum += val.age;
+     count++;
+     return { sum, count };
+     }, {
+        sum: 0, count: 0
+  });
+  return (sum / (count || 1));
+};
+console.log(getUserAverageAge(users));
+
+// 2 uzduotis
+
+const getUsersNames = () => users.map((user) => user.name);
```

Comment:
This function should accept array as an argument same as `getUserAverageAge` does.